boards/common-partitions.mk

```diff
@@ -0,0 +1,92 @@
+#
+# functions and definitions related to partitioning/formatting and using
+# diskimages
+#
+
+# The automatically created partition layout is always the same:
+# - part1, vfat, containing boot files and initrd
+# - part2, optional swap partition
+# - part3, optional linux partition.  At end of disk to allow expanding to fill
+#       the whole sdcard.  Could contain the root filesystem on a non-ramdisk
+#       image, or just bulk persistant storage for something in the ramdisk
```

Comment:
What are the plans to using `part2` and `part3`?

Reply:
just what is written there, really.  part2 is reserved for swap, part3 for bulk storage - the sun8i-v3s-licheepi-zero used part3 as its root filesystem, and we talked ages ago about using ipfs..